routes.js

```diff
@@ -69,6 +69,31 @@ module.exports = (app) => {
 
   // All routes below are authenticated
   router.use(passport.authenticate("jwt", { session: false }));
+
+  // All routes below need user permissions to access the restaurant
+  router.use(function (req, res, next) {
+    let userid = req.user.id;
+
+    User.findbypk(userid, {
+      include: [{ model: Restaurant, as: 'restaurants', where: { id: req.user.restaurantId } }]
+    }).then((user) => {
+
+      if (user.restaurants.length > 0) {
```

Comment:
We want to be checking if the user has access to the specific restaurant a user is trying to access, we aren't using the `user.restaurantId` field. 

So you should expect the `restaurantId` to come through the URL parameters, or try and access it based on whatever dish or menu is being accessed.  Let's chat about this tonight